app/models/User.coffee

```diff
@@ -180,6 +180,10 @@ module.exports = class User extends CocoModel
     return true if me.isAdmin()
     return true if me.hasSubscription()
     return false
+    
+  isEnrolled: ->
+    return true if @get('coursePrepaidID')
+    return false
```

Comment:
Trim down to:
`isEnrolled: -> @get('coursePrepaidID')`

If we want to make sure the return value is a boolean, then wrap with `Boolean`.